### ISSUE TITLE
Share link does not hold style settings

### ISSUE THREAD

**Issue by ThadeuAFMelo** [GoogleCloudPlatform/bigquery-geo-viz#44]
I tried to share some of my visualizations with my co-workers.
Unfortunately the styles are not being retained and they have to reconfigure.

Could there be a way to export those settings in a JSON or yaml file?
They could be reused online or offline.

**Comment by kits-ragh** [comment]
I also face the same issue, we should have exporting option for configuration just like open refine.

**Comment by delfrrr** [comment]
@ThadeuAFMelo @kits-ragh I don't think this project maintained, but you can try Dekart https://dekart.xyz/docs/about/playground/
